components/app.js

```diff
@@ -0,0 +1,26 @@
+class App {
+  constructor(gradeTable) {
+    this.handleGetGradesError = this.handleGetGradesError.bind(this);
+    this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+    this.gradeTable = gradeTable;
+  }
+  handleGetGradesError(error) {
+    console.error(error);
+  }
+  handleGetGradesSuccess(grades) {
+    this.gradeTable.updateGrades(grades);
+  }
+  getGrades() {
+    $.ajax({
+      method: "GET",
+      url: "http://sgt.lfzprototypes.com/api/grades",
+      data: "none",
```

Comment:
This is unnecessary, data is for sending body data in a request, you can remove that.